src/app/components/pet-form/pet-form.component.ts

```diff
@@ -47,26 +49,37 @@ export class PetFormComponent implements OnInit {
   petProfileForm!: FormGroup;
   showValidationErrors = false;
 
-  // Selection options for goal dropdown
-  goalSelection: SELECTION[] = [
-    { value: CommonConstants.MAINTAIN, viewValue: 'Maintain' },
-    { value: CommonConstants.LOSE, viewValue: 'Lose' },
-    { value: CommonConstants.GAIN, viewValue: 'Gain' },
-  ]
+  goalSelectOptions = [CommonConstants.MAINTAIN, CommonConstants.LOSE, CommonConstants.GAIN,]
```

Comment:
this seems odd, can goalSelectOptions be moved to CommonConstants? Perhaps I don't understand entirely how it works. Does goalSelectOptions get changed depending on certain conditions?